### PR TITLE
V25y9uZi Draft operator docs for service-brokers.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Note: The long list of assumptions can also serve as list of things to add to the docs.

### CODE REVIEW

**Review by mook-as — COMMENTED** [review]
The _Automatic database setup_ option should probably not be available for non-developers, because they are based on the official docker images for mysql/postgres, which means we need to track down licensing.

Or we need to rewrite those images to be based on things we ship via SLE…

**Comment by andreas-kupries** [comment]
Yes. I saw the comments in the publishing ticket. Lets discuss more tomorrow.

**Comment by andreas-kupries** [comment]
Updated, `AUTO` removed.

File: Service-Broker-Setup.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+```
+
+# Deploying the chart in three steps
+
+Please review the previous section and ensure that all the assumptions
+are satisfied.
+
+## CAP location and namespaces
+
+Determine the namespaces used for the CF and UAA roles in his CAP
```
Comment (mook-as) [inline Service-Broker-Setup.md]: Up to now, the user was genderless (not sure if that was intentional). Yes, this is a silly nit and I wouldn't block merging for it :D

File: Service-Broker-Setup.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  depending on the setup of the database. See also
+  [Appendix II: Chart variables](#appendix-ii-chart-variables) for
+  more information on the available values.
+
+# Appendix II: Chart variables
+
+|Variable			|Meaning|
+|---				|---|
+|env.SERVICE_LOCATION		|Broker location as seen by CAP cluster|
+|env.SERVICE_<db>_HOST		|Host the database lives on|
```
Comment (mook-as) [inline Service-Broker-Setup.md]: It appears that GitHub doesn't render `<db>` like literal text, but instead it shows up as an HTML tag (and therefore invisible).

**Review by mook-as — CHANGES_REQUESTED** [review]
Seems generally good, just a couple tiny nitpicks

File: Service-Broker-Setup.md (hunk 1 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md]: Subconscious semi-intentional I believe. Will fix.

File: Service-Broker-Setup.md (hunk 2 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md]: :( Ok. Hm. Does `(db)` work ?

Comment (mook-as) [reply Service-Broker-Setup.md]: Yeah, that should be fine, I think.

File: Service-Broker-Setup.md (hunk 1 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md]: Addressed with commit b951c6adc7d32f03d6f9a6ed411d7690ded72b7d

File: Service-Broker-Setup.md (hunk 2 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md]: Addressed with commit b951c6adc7d32f03d6f9a6ed411d7690ded72b7d

File: Service-Broker-Setup.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+# Appendix II: Chart variables
+
+|Variable			|Meaning|
+|---				|---|
+|env.SERVICE_LOCATION		|Broker location as seen by CAP cluster|
+|env.SERVICE_(db)_HOST		|Host the database lives on|
+|env.SERVICE_(db)_PORT		|Port the database listens on|
+|env.SERVICE_(db)_USER		|User name for database connections|
+|env.SERVICE_(db)_PASS		|Password for database connections|
+|env.SERVICE_POSTGRESQL_SSLMODE	|Connection to postgres server, one of `disable`, `require`, `verify-ca`, `verify-full`|
```
Comment (mook-as) [inline Service-Broker-Setup.md]: Actually, maybe we should just link to https://godoc.org/github.com/lib/pq?

File: Service-Broker-Setup.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+* The user knows how to get the service broker charts, or has them
+  already. This also implies that the user knows the location of the
+  charts in the filesystem.
+
+* The user has stored the path to the chart in the environment
+  variable `CHART`.
+
+  ```
+  CHART=/path/to/mysql-chart
```
Comment (mook-as) [inline Service-Broker-Setup.md:29]: What happens if it's a published chart?

**Review by mook-as — APPROVED** [review]

File: Service-Broker-Setup.md (hunk 3 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md]: +1, will do.

File: Service-Broker-Setup.md (hunk 4 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md:29]: See line 21-23.

File: Service-Broker-Setup.md (hunk 3 above)
Comment (andreas-kupries) [reply Service-Broker-Setup.md]: Addressed with commit 0293cbde36258e3f6070d265af2180394c260697

File: Service-Broker-Setup.md (hunk 4 above)
Comment (mook-as) [reply Service-Broker-Setup.md:29]: Right, but in the published chart case it wouldn't be on the filesystem.

Meh, close enough, will merge :D

**Pull request merged**